src/main.js

```diff
@@ -33,6 +33,11 @@ app.setPath('userData', userDataPath);
 // Update cwd based on environment and platform
 setCurrentWorkingDirectory();
 
+// Register custom schemes with privileges
+protocol.registerSchemesAsPrivileged([
```

Comment:
@deepak1556 is that to make https://github.com/Microsoft/vscode/blob/a6ee65e647e6eeb1b6926d4100276e7afff14510/src/vs/code/electron-main/app.ts#L746 work? Should add a todo that we can remove this once we move to a different solution.